docs/usage/commands/k3d_image_import.md

```diff
@@ -39,6 +40,25 @@ k3d image import [IMAGE | ARCHIVE [IMAGE | ARCHIVE...]] [flags]
       --verbose      Enable verbose output (debug logging)
 ```
 
+
+### Loading modes
+
+#### Auto
+
+Auto-determine whether to use `direct` or `tools-node`.
+
+For remote container runtimes, `tools-node` is faster due to less network overhead, thus it is automatically selected for remote runtimes.
+
+Otherwise direct is used.
+
+#### Direct
+
+Directly load the given images to the k3s nodes. No separate container is spawned, no intermediate files are written.
+
+#### Tools Node
+
+Start a `k3d-tools` container in the container runtime, copy images to that runtime, then load the images to k3s nodes from there.
+
 ### SEE ALSO
```

Comment:
This is an auto-generated file (generated by the CLI framework cobra), so this will likely get overwritten upon the next refresh of the documentation.
Feel free to add another page to the usage docs, e.g. `docs/usage/importing_images.md` and add a link to the command's helptext, so it will also be included in this auto-generated doc :+1: